wraith/core.yaml

```diff
@@ -0,0 +1,187 @@
+# The engine to run Wraith with.
+browser: "casperjs"
+
+# The domain to take screenshots of.
+domains:
+  # glowfic: "http://web:3000" # Docker friendly
```

Comment:
Is there any way to have them be actually different domain names, like dev: and docker:, or is there no way to specify and Wraith will just get confused?

Reply:
afaict wraith will assume you want to compare the two domains